go.mod

```diff
@@ -14,5 +14,6 @@ require (
 	golang.org/x/sys v0.0.0-20210124154548-22da62e12c0c // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 	k8s.io/apimachinery v0.20.2
+	k8s.io/utils v0.0.0-20210305010621-2afb4311ab10
```

Comment:
this is at least not a staging repo right? but also no versioned tags?

Reply:
we appear to only use two small methods. we should consider just copying those into an `internal` package.